README.md

```diff
@@ -1,32 +1,47 @@
-# smooth-operator
-Shared logic and structure library for PDOK operators. 
-This repo contains global Kinds and shared code base for other operator.
-
 <p align="center">
     <a href="https://youtu.be/4TYv2PhG89A?si=X_a7zpjaD_esgUAO&t=74">
         <img src="docs/mascotte.png" alt="Smooth-operator" title="Smooth-operator" width="300" />
     </a>
 </p>
 
+# Smooth Operator
```

Comment:
Zou je de titel naar Smooth Operator Library willen wijzigen. Denk dat het belangrijk is dat we aangeven dat dit een lib is en géén operator.